code/modules/power/supermatter/supermatter.dm

```diff
@@ -355,6 +355,8 @@
 /obj/machinery/power/supermatter/proc/Consume(atom/movable/AM)
 	if(istype(AM, /mob/living))
 		var/mob/living/user = AM
+		if(user.ckey)
+			log_attack("[src] has consumed [pulledby ? "(being pulled by [pulledby])" : "last touched by [fingerprintslast]"] [AM]/([key_name(user)])")
```

Comment:
I would imagine it just prints NULL? I couldn't really test this since I don't have a database on my test serber.